src/net/java/sip/communicator/plugin/jabberaccregwizz/IceConfigPanel.java

```diff
@@ -202,14 +205,27 @@ public void actionPerformed(ActionEvent e)
 
                 if (stunServer != null)
                 {
+                    String userName;
+                    String password;
+
+                    try
+                    {
+                        userName = new String(
+                            stunServer.getUsername(), StandardCharsets.UTF_8.name());
+                        password = new String(
+                            stunServer.getPassword(), StandardCharsets.UTF_8.name());
+                    }
+                    catch (UnsupportedEncodingException ex) {
```

Comment:
Formatting.
As I said, 'if it throws, use StringUtils.toEncodedString', so why don't you?

Reply:
Could not resolve symbol